### PR TITLE
linkify messages so you can follow links

### PR DESCRIPTION
**Pull request by Connoropolous**
@levity are there any side effects of doing this that I might not know about? I looked under the hood at some of the code, but there's actually a lot going on there. 

fixes: https://trello.com/c/ui9Q3pMu/114-2-linkify-urls-in-direct-message-threads

### CODE REVIEW

**Comment by levity** [comment]
This wraps code in `<p>` tags, which will mess up the layout. You could change the CSS so that it still looks the same... remember to check the spacing between two consecutive messages from the same person.

**Comment by Connoropolous** [comment]
@levity thanks for that. I've added an option to use present without wrapping in p tags (but it defaults to that behaviour.

File: src/util/text.js [hunk 1]
```diff
@@ -17,13 +17,13 @@ export const truncate = (text, length) =>
 export function present (text, opts = {}) {
   if (!text) return ''
 
-  // wrap in a <p> tag
-  if (text.substring(0, 3) !== '<p>') text = `<p>${text}</p>`
+  // wrap in a <p> tag, do this be default, require opt out
```
Comment (levity) [inline src/util/text.js]: s/be/by

**Review by levity — APPROVED** [review]
looks good once you fix the typo

**Pull request merged**